### PR TITLE
JSA-5: Add New Loan Component

### PR DESCRIPTION
**Pull request by amelieykw**

### CODE REVIEW

File: front-end/NewLoan/View.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+                    <TextInput style={styles.input} placeholder='5 years fixed'></TextInput>
+                </View>
+                <View style={styles.Description} >
+                    <Icon name='contacts' size={30} color='gray' containerStyle={styles.DescriptionContactsIcon} />
+                    <View style={styles.DescriptionLabel}>
+                        <Text>3% yearly interest rate</Text>
+                        <Text>Fixed low interest rate for students only</Text>
+                    </View>
+                    <Icon name='info' size={30} color='gray' containerStyle={styles.DescriptionInfoIcon} />
+                </View>
```
Comment (adamgyulavari) [inline front-end/NewLoan/View.js]: @amelieykw these two should be a product details component

File: front-end/NewLoan/View.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                <View style={styles.MonthlyPaid}>
+                    <Text style={styles.MonthlyPaidData}>$ 86.84</Text>
+                    <Text style={styles.MonthlyPaidLabel}>paid monthly</Text>
+                </View>
+                <View style={styles.InterestPaid}>
+                    <Text style={styles.InterestPaidData}>$ 390.61</Text>
+                    <Text style={styles.InterestPaidLabel}>interest paid altogether</Text>
+                </View>
```
Comment (adamgyulavari) [inline front-end/NewLoan/View.js]: @amelieykw these 2 should be payment details component

File: front-end/NewLoan/View.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
```
Comment (adamgyulavari) [inline front-end/NewLoan/View.js]: @amelieykw the file name should match the component's name

File: front-end/NewLoan/View.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    AppRegistry,
+    StyleSheet,
+    Text,
+    TextInput,
+    View,
+    Platform,
+} from 'react-native';
+import { Header, Button, Icon } from 'react-native-elements';
+
+export default class NewLoan extends Component {
```
Comment (adamgyulavari) [inline front-end/NewLoan/View.js]: @amelieykw use function component instead

File: front-end/NewLoan/View.js [hunk 5]
```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    TextInput,
+    View,
+    Platform,
+} from 'react-native';
+import { Header, Button, Icon } from 'react-native-elements';
```
Comment (adamgyulavari) [inline front-end/NewLoan/View.js]: @amelieykw use this library instead, this is what all the projects will use: https://docs.nativebase.io/

Comment (amelieykw) [reply front-end/NewLoan/View.js]: ? I asked you this morning in slack. Can we use this one https://react-native-elements.github.io/react-native-elements/? You said that we can.

Which one exactly should we use?

**Pull request closed without merging**